core/frontend/src/MarginPercent.ts

```diff
@@ -7,8 +7,14 @@
  */
 
 import { Geometry } from "@itwin/core-geometry";
-/** Margins for white space to be left around view volumes for [[ViewState.lookAtVolume]].
- * Values mean "fraction of view size" and must be between 0 and .25.
+
+/** Specifies margins to apply around a view volume for methods like [[ViewState.lookAtVolume]] and [[Viewport.zoomToElements]], expanding the
```

Comment:
might be worth pointing out here that this only applies in 2d or if the camera is off